### PR TITLE
PWGGA/GammaConv: GammaIsoTree changed MC iso definition and default cuts

### PR DESCRIPTION
**Pull request by fjonasALICE**

### CODE REVIEW

File: PWGGA/GammaConvBase/AliCaloPhotonCuts.h [hunk 1]
```diff
@@ -703,7 +709,7 @@ class AliCaloPhotonCuts : public AliAnalysisCuts {
 
   private:
 
-    ClassDef(AliCaloPhotonCuts,108)
+    ClassDef(AliCaloPhotonCuts,109)
```
Comment (dmuhlhei) [inline PWGGA/GammaConvBase/AliCaloPhotonCuts.h:712]: no classdef increment needed

File: PWGGA/GammaConvBase/AliCaloPhotonCuts.h [hunk 2]
```diff
@@ -448,7 +448,13 @@ class AliCaloPhotonCuts : public AliAnalysisCuts {
     void        SetExoticsQA(Bool_t enable)                     { fDoExoticsQA         = enable; return;}
 
     Float_t     GetMinClusterEnergy()                           { return fMinEnergy;};
-
+    
+    Double_t    GetMinEtaCut() {return fMinEtaCut;};
```
Comment (dmuhlhei) [inline PWGGA/GammaConvBase/AliCaloPhotonCuts.h:452]: superfluous ';'

File: PWGGA/GammaConvBase/AliCaloPhotonCuts.cxx [hunk 3]
```diff
@@ -6318,6 +6318,11 @@ Bool_t AliCaloPhotonCuts::SetNLM(Int_t nlm)
     fMinNLM =3;
     fMaxNLM =100;
     break;
+  case 5: // NLM < 3
```
Comment (dmuhlhei) [inline PWGGA/GammaConvBase/AliCaloPhotonCuts.cxx:6321]: what is the difference to case 3?

File: PWGGA/GammaConv/AliAnalysisTaskGammaIsoTree.cxx [hunk 4]
```diff
@@ -3775,6 +3853,11 @@ void AliAnalysisTaskGammaIsoTree::FillCaloHistosPurity(AliAODCaloCluster* clus,v
   
   Double_t m02 = clus->GetM02();
 
+  if(m02 == 0 || clus->GetNCells() == 0) {
```
Comment (dmuhlhei) [inline PWGGA/GammaConv/AliAnalysisTaskGammaIsoTree.cxx:3856]: we should omit these cout running on grid, shouldn't we?

**Comment by fjonasALICE** [comment]
thanks for the comments Daniel, should be all fixed now!

**Comment by dmuhlhei** [comment]
+1

**Pull request merged**